nixos/modules/tasks/filesystems/exfat.nix

```diff
@@ -1,11 +1,13 @@
-{ config, lib, pkgs, ... }:
+{ config, lib, pkgs, kernel, ... }:
 
 with lib;
 
 {
   config = mkIf (any (fs: fs == "exfat") config.boot.supportedFilesystems) {
-
-    system.fsPackages = [ pkgs.exfat ];
-
+    system.fsPackages = (if lib.versionOlder "5.7" then [
```

Comment:
`lib.versionOlder` needs two arguments.

Reply:
This'll teach me for programming tired... Will fix shortly